python/src/wslink/chunking.py

```diff
@@ -87,7 +87,7 @@ class UnChunker:
 
     def __init__(self):
         self.pending_messages = {}
-        self.max_message_size = 512
+        self.max_message_size = 3000
```

Comment:
can we replace that with the following instead? (also add `import os` at the top)

```python
def __init__(self):
   self.pending_messages = {}
   self.max_message_size = int(os.environ.get("WSLINK_AUTH_MSG_SIZE", 512))
```

Reply:
That way, for your usecase, you just need to set that ENV variable to your expected size.